app/models/concerns/is_an_entity_imported_from_feeds.rb

```diff
@@ -37,5 +38,26 @@ module IsAnEntityImportedFromFeeds
       where(id: self.all.select(:id).pluck(:id) - self.where_imported_from_active_feed_version.select(:id).pluck(:id))
     }
 
+    attr_accessor :add_feed_versions, :remove_feed_versions
```

Comment:
Could you make the names of these two virtual attributes a bit more descriptive? Perhaps `imported_from_feed_versions` and `not_imported_from_feed_versions?